cypress/e2e/seededFlows/articleFlows/commentModerationTools.spec.js

```diff
@@ -30,83 +23,84 @@ describe('Moderation Tools for Comments', () => {
 
       cy.get('@trustedUser').then((trustedUser) => {
         cy.loginAndVisit(trustedUser, '/series_user/series-test-article-slug');
+        // Reference: https://filiphric.com/how-to-wait-for-page-to-load-in-cypress
```

Comment:
Typically, I would provide a link only to a solution to an edge case issue/fix. I don't think that's this case and would rather omit it.
```suggestion
```